doc/admin-guide/files/records.config.en.rst

```diff
@@ -4269,6 +4269,12 @@ HTTP/2 Configuration
    frames. Write operation will be triggered at least once every this configured
    number of millisecond regardless of pending data size.
 
+.. ts:cv:: CONFIG proxy.config.http2.default_buffer_water_mark INT 32768
+   :reloadable:
+   :units: bytes
+
+   Specifies the buffer water mark size in bytes in the HTTP/2 layer.
```

Comment:
I would like to see a better description.  One like: https://docs.trafficserver.apache.org/admin-guide/files/records.config.en.html#proxy-config-http-default-buffer-water-mark

Reply:
Updated the doc blurb for this. 

There will be more docs coming to describe the why of this (in a subsequent PR)